src/index.html

```diff
@@ -37,32 +37,38 @@ <h2 class="guest-nav-cost">You have spent $1,000 on your stays so far!</h2>
     <section class="book-room-view">
       <div class="date-view">
         <h3 class="date-view-title">Make a Booking</h3>
-        <div class="date-view-input">
+        <div class="date-view-input" id="searchSection">
           <input aria-label="select-date-for-booking" type="date" id="dateInput" class="date-input" min="2021-03-05"
-            max="2025-12-31" required>
```

Comment:
I would consider adding aria-required = "false" here and then toggling it to true in your index.js to help with accessibility